tests/operators/glob.cpp

```diff
@@ -78,19 +78,19 @@ TEST_CASE("Glob") {
     {
         auto rows = storage.select(glob(&Employee::firstName, "S*"));
         REQUIRE(rows.size() == 9);
-        auto trueValuesCount = std::count(rows.begin(), rows.end(), true);
+        auto trueValuesCount = count(rows.begin(), rows.end(), true);
         REQUIRE(trueValuesCount == 3);
     }
     {
-        auto rows = storage.select(glob(distinct(&Employee::firstName), "S*"));
+        auto rows = storage.select(distinct(glob(&Employee::firstName, "S*")));
         REQUIRE(rows.size() == 2);
-        auto trueValuesCount = std::count(rows.begin(), rows.end(), true);
```

Comment:
why did you remove `std::`?

Reply:
ADL. It's the same with `find_if` somes lines above that you used.